processor/spanmetricsprocessor/processor.go

```diff
@@ -223,6 +252,10 @@ func (p *processorImp) Start(ctx context.Context, host component.Host) error {
 		return fmt.Errorf("failed to find metrics exporter: '%s'; please configure metrics_exporter from one of: %+v",
 			p.config.MetricsExporter, availableMetricsExporters)
 	}
+
+	if err := view.Register(metricViews()...); err != nil {
+		log.Fatalf(err.Error())
+	}
```

Comment:
This will cause an error if someone were to add two spanmetrics processors to a collector. It should be done in an `init` function.